src/ray/raylet/node_manager.cc

```diff
@@ -953,12 +953,11 @@ void NodeManager::HandleActorStateTransition(const ActorID &actor_id,
       }
     }
   }
-  RAY_LOG(DEBUG) << "Actor notification received: actor_id = " << actor_id
-                 << ", node_manager_id = " << actor_registration.GetNodeManagerId()
-                 << ", state = "
-                 << ActorTableData::ActorState_Name(actor_registration.GetState())
-                 << ", remaining_restarts = "
-                 << actor_registration.GetRemainingRestarts();
+  RAY_LOG(INFO) << "Actor notification received: actor_id = " << actor_id
```

Comment:
I am afraid this will spam node manager. Why does it have to be INFO?

Reply:
fixed